app/src/main/java/com/asfoundation/wallet/ui/OneStepPaymentReceiver.java

```diff
@@ -93,13 +82,6 @@ private void startOneStepTransfer(TransactionBuilder transaction, boolean isBds)
     walletCreationText = null;
   }
 
-  private Single<Wallet> createWallet() {
-    showLoadingAnimation();
-    return paymentReceiverInteract.createWallet()
-        .observeOn(AndroidSchedulers.mainThread())
-        .doAfterTerminate(this::endAnimation);
-  }
-
   private void endAnimation() {
     walletCreationAnimation.setVisibility(View.INVISIBLE);
     walletCreationCard.setVisibility(View.INVISIBLE);
```

Comment:
method **showLoadingAnimation** is not used and can be removed

Reply:
Logic missing to use of the animation. Fixed now.